Kudu.Services/Diagnostics/LinuxProcessController.cs

```diff
@@ -15,7 +21,35 @@ namespace Kudu.Services.Performance
 
     public class LinuxProcessController : Controller
     {
+        const string dotnetMonitorPort = "50051";
+        const string DotNetMonitorAddressCacheKey = "DotNetMonitorAddressCacheKey";
+        const string AcceptEncodingHeader = "Accept-Encoding";
+
+        private readonly IMemoryCache _cache;
```

Comment:
Do we need to cache the address for DOTNET Monitor? Since these requests would be fairly low, why not simply read the file to determine the address for a request?

Reply:
Just doing that to avoid any File IO if someone has Kudu Console opened up and are playing around a lot. If you think this is not going to help, I can remove it.

Reply:
I think let's remove it, we can add the cache if it's needed later